scripts/sizeSnapshot/webpack.config.js

```diff
@@ -186,7 +201,10 @@ function createWebpackConfig(entry, environment) {
         reportFilename: `${entry.id}.html`,
       }),
     ],
-    entry: { [entry.id]: path.join(workspaceRoot, entry.path) },
+    context: __dirname,
+    entry: {
+      [entry.id]: `./index.js!=!data:text/javascript,import ${importNames} from '${entry.import}';console.log(foo);`,
```

Comment:
This is the magic sauce: context + file name + data url.

Reply:
IMO this deserves a comment in code. I don't think this syntax is widely known (I just learned about this and it's pretty hard to search for it).

Reply:
You are right, I've added some links in a comment